presto-main/src/test/java/com/facebook/presto/type/TestRealOperators.java

```diff
@@ -186,6 +187,16 @@ public void testCastToVarchar()
     }
 
     @Test
+    public void testCastToChar()
+            throws Exception
+    {
+        assertFunction("CAST(REAL'754.1985' as CHAR)", CHAR, "754.1985");
+        assertFunction("CAST(REAL'-754.2008' as CHAR)", CHAR, "-754.2008");
```

Comment:
There should also be tests that truncate values:
`CAST(REAL'-754.2008' as CHAR(5)` should fail (exception: truncates value).

Ditto for other data types.